operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -77,6 +79,20 @@ func CreateOrUpdateLokiStack(
 		}
 	}
 
+	storageSchemas := storage.UpdateSchemas(
+		time.Now().UTC(),
+		stack.Spec.Storage.SchemaVersion,
+		stack.Status.Storage.Schemas,
+	)
+
+	err = status.SetStorageSchemaStatus(ctx, k, req, storageSchemas)
+	if err != nil {
+		ll.Error(err, "failed to set storage schema status")
+		return err
+	}
```

Comment:
The reason I put the status update before the `createOrUpdate` method is because I wanted to avoid a scenario where, for some reason, one resource fails the push and thus the entire function exits. If the `configmap` got updated, then we might have some sync issues between the `configmap` and the `status`.